test/unit/components/split-content.test.js

```diff
@@ -0,0 +1,14 @@
+const expectComponent = require('../component-helper').expectComponent;
+
+describe('Split content component', () => {
+  it('should render', () => {
+    expectComponent(
+      'split-content',
+      {
+        children: [],
+      },
+      `<div class="panel">
+      </div>`
+    );
+  });
```

Comment:
I tried to leave children out of it as much as possible unless the component's render depended on having children. In theory how the child renders is tested with each component individually and in frontend we don't care what the children of a component are.